src/server.js

```diff
@@ -26,7 +26,15 @@ const buildHtmlString = (body, cache) => {
     `<!DOCTYPE html>
       <html>
         <head>
-          <title>Hello World</title>
+          <title>The Gazelle</title>
+          <link rel="stylesheet" type="text/css" href="/build/main.css">
+          <link rel="apple-touch-icon" sizes="180x180" href="/static/favicons/apple-touch-icon.png">
```

Comment:
That path of all of these is wrong. It should not start with /static. The correct path is:

```
href="/favicons/apple-touch-icon.png"
```

when a route matches, node automatically tries to look for it in static before passing it along.